drivers/start_date_dq/app/views/start_date_dq/warehouse_reports/reports/index.haml

```diff
@@ -25,6 +25,7 @@
         - @enrollments.each do |row|
           - client = row.client
           - values = @report.column_values(row)
+          - values[0] = number_with_delimiter(values[0]) # Add comma separator to days difference
```

Comment:
So, this makes me nervous since it's position based, but the data is fetched really far away.  Could we maybe change the return value of `@report.column_values(row)` into a hash?  then you could check for a key and make a determination on how to display based on that key.

Reply:
Yeah this is what I didn't like about having it far away in the report.. I'll change it to a dict, but then the view still needs to know about the key names. I agree it's better than indices though.

Reply:
I'm fine with the view knowing which key names it needs to present differently, though sometimes we push that back to the model `@report.display_value(key, value)` or something like that.  I was tempted to suggest handling that within the `column_values` method, but with the export we don't want to manipulate the data since Excel handles the display just fine.

Reply:
This is fixed